test/new-e2e/pkg/utils/infra/stack_manager.go

```diff
@@ -318,26 +318,29 @@ func (sm *StackManager) getStack(ctx context.Context, name string, config runner
 		if err == nil {
 			break
 		}
-		if retryStrategy := sm.getRetryStrategyFrom(err); retryStrategy != noRetry {
-			fmt.Fprintf(logger, "Got error that should be retried during stack up, retrying with %s strategy", retryStrategy)
-			err := sendEventToDatadog(fmt.Sprintf("[E2E] Stack %s : retrying Pulumi stack up", name), err.Error(), []string{"operation:up", fmt.Sprintf("retry:%s", retryStrategy)}, logger)
+
+		retryStrategy := sm.getRetryStrategyFrom(err)
```

Comment:
Sorry if this is a stupid question but I'm confused. Shouldn't we switch the default return value to `reUp` there? https://github.com/DataDog/datadog-agent/blob/ea3b9a4214b178a94a4962a231a1c408c296f5bc/test/new-e2e/pkg/utils/infra/stack_manager.go#L398

Reply:
Oh my, forgot to commit the change.Thank you for the catch!